bin/create-cluster.sh

```diff
@@ -8,6 +8,10 @@
 
 . ../etc/gke-env.cfg
 
+GKE_NETWORK_NAME="${GKE_NETWORK_NAME:-default}"
```

Comment:
Hey, I noticed the default network on engineering-docs is Legacy. Should I have a network configuration like that on engineering-docs?